packages/generator/src/generator.spec.ts

```diff
@@ -138,7 +142,7 @@ describe('generator', () => {
     });
 
     it('generates expected number of files', () => {
-      expect(files.length).toBe(35);
+      expect(files.length).toBe(34);
```

Comment:
not sure why those numbers changed.. is this really intended?

Reply:
the options-per-service are not part of the TS-morph project anymore by geneated via `fs`